packages/opentelemetry-types/src/trace/tracer.ts

```diff
@@ -88,3 +93,24 @@ export interface Tracer {
    */
   getHttpTextFormat(): HttpTextFormat;
 }
+
+/**
+ * TracerConfig provides an interface for configuring a Tracer.
+ */
+export interface TracerConfig {
```

Comment:
Looks like this is same as the `BasicTracerConfig` interface in `basic-tracer` package?

Reply:
ooups, rebase issue, sry